Sources/MapboxMaps/Foundation/WeakSet.swift

```diff
@@ -1,22 +1,22 @@
 import Foundation
 
 // swiftlint:disable force_cast
-internal class WeakCameraAnimatorSet {
+internal class WeakSet<T> {
```

Comment:
Nice, I'd tried to do something like this earlier but was running into some weird issue with it.